tests/data/dummy_aea/connections/local/connection.py

```diff
@@ -0,0 +1,365 @@
+# -*- coding: utf-8 -*-
```

Comment:
Ideally we would copy these in at runtime of the tests and remove again at the end. Otherwise they can get out of sync with the actual implementations. @marcofavorito  thoughts?

Reply:
Agreed. Even better, they could be symbolic links to the modules in `aea`.

Reply:
I tried symbolic links but then our import functionality doesn;t play with it..